client/js/gdpr/chart.js

```diff
@@ -114,8 +115,29 @@ export class GdprChart extends HTMLElement {
     this.clearHover();
   }
 
-  onScroll() {
-    this.clearHover();
+  onMouseWheel(ev) {
+    if (ev.ctrlKey) {
+      const {min, max, pow} = Math;
+      const scale = pow(10, ev.deltaY / 100);
+
+      const finalZoom = min(max(this.zoom * scale, 0.1), 3);
```

Comment:
Mouse wheel delta is a positive value or a negative value depending on which way the user is scrolling, but it's also linear. I need to use a logarithm in order for the scaling to feel linear when you scroll (i.e., scrolling one tick inwards should move you from 0.5x to 0.4x, but having the same movement scroll you from 1.5x to 1.6x would feel inconsistent). Finally, the min and max are so that the user can't scroll too far in either direction.